src/util/bv_encoding.h

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************\
+
+Module: bv_encoding configuration
+
+Author: Diffblue Ltd
+
+\*******************************************************************/
```

Comment:
I think I mentioned this before but we want to move away from this style of module header... we might have to adjust our linter for that though.

Reply:
I have removed this commit from this PR.